packages/smooth_app/lib/data_models/continuous_scan_model.dart

```diff
@@ -237,6 +237,15 @@ class ContinuousScanModel with ChangeNotifier {
     await refresh();
   }
 
+  Future<void> removeBarcode(
+    final String barcode,
+  ) async {
+    await _daoProductList.pop(productList, barcode);
+    await _daoProductList.pop(_history, barcode);
```

Comment:
I don't think you should remove it from the history too.
The idea is just to un-clutter the scan carousel, not the whole history.

Reply:
Changes done.